### PR TITLE
Multiple Creatives

### PR DESCRIPTION
**Pull request by allentsai**
This commit makes FirehoundBlob contain multiple creatives instead of a single
one.

### CODE REVIEW

File: src/PaperG/FirehoundBlob/FirehoundBlob.php [hunk 1]
```diff
@@ -143,48 +145,72 @@ public function toAssociativeArray()
...
-            self::VERSION            => self::CURRENT_VERSION,
+            self::NAME => isset($this->name) ? $this->name : null,
+            self::IDENTIFIER => isset($this->identifier) ? $this->identifier : null,
+            self::START_DATE => isset($this->startDate) ? $this->startDate : null,
+            self::END_DATE => isset($this->endDate) ? $this->endDate : null,
+            self::BUDGETS => isset($this->budgets) ? $budgets : null,
+            self::TARGETING => isset($this->targeting) ? $this->targeting->toAssociativeArray() : null,
+            self::CONTEXT => isset($this->context) ? $this->context->toAssociativeArray() : null,
+            self::PLATFORM_TARGETING => isset($this->platformTargeting) ? $this->platformTargeting->toAssociativeArray(
+                ) : null,
```
Comment (zhiachong) [inline src/PaperG/FirehoundBlob/FirehoundBlob.php]: Odd formatting here

Comment (allentsai) [inline src/PaperG/FirehoundBlob/FirehoundBlob.php]: Yeah, don't know why they decided that was the right way to format....

**Comment by a37wong** [comment]
Okay this is pretty good, I'm wondering if you want to have at least one integration test that uses your new versions.

Also I'm unsure if the transformer works fine considering you upped the version on FirehoundBlob but did not change the transformer code (which relies on the version).  Maybe it works but maybe it doesn't.

Also, at this point, I think it may be worthwhile to update the readme.md to link off to the architecture doc and api doc so that someone interested in what the latest JSON format is can just go to the repo on github and figure it out.

**Comment by allentsai** [comment]
@a37wong Because we version releases, the transformer will not use the new version of FirehoundBlob until FH/PL is updated.  The PR that updates PL/FH to use what will be the latest release (we're going to v1.0.0) will also have to contain the update to the transformer.

**Comment by allentsai** [comment]
I'll update the readme in a separate PR.

**Comment by a37wong** [comment]
Okay, that makes to me.  I'm wondering if we should later shift the transformer to this library and have tests run against it here?

**Comment by allentsai** [comment]
@a37wong Yeah, we probably will want to do that, but there are a lot of seams that need to be cut.  Mostly because PackagedCampaign is so tightly coupled.  We might need to create an interface to abstract it out a little.

**Pull request merged**